distributed/tests/test_worker.py

```diff
@@ -30,7 +30,6 @@
 from distributed.utils_test import (inc, mul, gen_cluster, div, dec,
                                     slow, slowinc, gen_test, cluster,
                                     captured_logger)
-from distributed.utils_test import loop # flake8: noqa
```

Comment:
It looks like this change is causing test failures

Reply:
Indeed, I must have deleted this line by mistake.